relayer/relays/beacon/header/header.go

```diff
@@ -73,7 +75,7 @@ func (h *Header) Sync(ctx context.Context, eg *errgroup.Group) error {
 
 	log.Info("starting to sync finalized headers")
 
-	ticker := time.NewTicker(time.Second * 10)
+	ticker := time.NewTicker(time.Minute * 2)
```

Comment:
Will this interval too long for local setup?

Reply:
I don't think so, no, because a finalized header will be [synced every 316 blocks](https://github.com/Snowfork/snowbridge/pull/1206/files#diff-92cc554d5b7218e7576839ca25a3b71ae0bbfa8200552beaf703a95a1a0da2a3R554) (approximately 6 times in a sync committee period).